src/server/zset_family.cc

```diff
@@ -2131,8 +2133,8 @@ void ZSetFamily::Register(CommandRegistry* registry) {
             << CI{"ZINCRBY", CO::FAST | CO::WRITE | CO::DENYOOM, 4, 1, 1, 1}.HFUNC(ZIncrBy)
             << CI{"ZINTERSTORE", kStoreMask, -4, 3, 3, 1}.HFUNC(ZInterStore)
             << CI{"ZLEXCOUNT", CO::READONLY, 4, 1, 1, 1}.HFUNC(ZLexCount)
-            << CI{"ZPOPMAX", CO::READONLY, 3, 1, 1, 1}.HFUNC(ZPopMax)
-            << CI{"ZPOPMIN", CO::READONLY, 3, 1, 1, 1}.HFUNC(ZPopMin)
+            << CI{"ZPOPMAX", CO::READONLY, -2, 1, 1, 1}.HFUNC(ZPopMax)
```

Comment:
`CO::READONLY` is incorrect. Should this be `CO::WRITE | CO::FAST`?

Reply:
Welcome back @alisaifee ! 
Yes, it's a copy paste bug  🤷🏼 . Please change to WRITE semantics.

Reply:
Addressed with the latest (forced) push!